exonum/src/runtime/dispatcher/schema.rs

```diff
@@ -149,17 +149,54 @@ impl Schema<&Fork> {
             return Err(Error::ServiceIdExists);
         }
 
-        let id = spec.id;
-        let name = spec.name.clone();
+        let instance_id = spec.id;
+        let instance_name = spec.name.clone();
+        let pending_status = InstanceStatus::Active;
```

Comment:
Micro-nit: You can write here `Some(InstanceStatus::Active)` to have a bit clearer constructor call below.

Reply:
Good catch, but `pending_status` is used in the method `put` on the line 164. `Some(..)` vs `unwrap()` ? 😄

Reply:
Ah, indeed. Those sneaky `Copy` types got me again :male_detective: